src/main/java/com/alysaa/geyseradmintools/Gat.java

```diff
@@ -14,19 +19,26 @@
 import java.util.logging.Logger;
 
 public class Gat extends JavaPlugin {
+
     public static Gat plugin;
+    public static Logger logger;
+
     @Override
     public void onEnable(){
         new Metrics(this, 10943);
         plugin = this;
+        logger = getLogger();
+
         createFiles();
         checkConfigVer();
+        ItemStackFactory.createStarTool();
```

Comment:
this must be called on startup to create the static item; it can be accessed later